### DISCUSSION TITLE
Расширение маркдауна

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/content#1949] [Идеи] [answered]
Сейчас мы используем стандартные markdown-теги, для работы в контенте. 

Бывают ситуации, когда их не хватает, и markdown следует расширить. 

В этой дискуссии можно проголосовать за сохранение стандартного markdown'а, или указать, каких элементов вам не хватает.

**Comment by furtivite** [comment]
Вариант добавления ссылки с открытием в новой вкладке

```
[Подробнее о HEX-цветах в Доке](https://doka.guide/css/web-colors/#hex){target="_blank"}
```

**Reply by furtivite** [reply]
В теории, таким образом можно передавать практически любой код, который находится внутри тега, даже класс или инлайновые стили, так что надо бы следить за таким.

**Comment by pepelsbey** [comment, marked as answer]
Подпись к картинке.

Её правильно разметить как figure/figcaption. Поэтому предлагаю:

```
![]()
Text
```

Превращать в

```
<figure>
  <img>
  <figcaption>Text</figcaption>
</figure>
```

А вот такое — нет, текст останется абзацем:

```
![]()

Text
```

**Reply by solarrust** [reply]
А что на счёт добавления атрибутов к картинке? Как в вопросе Егора.

**Reply by pepelsbey** [reply]
Я бы постарался по максмуму свести всё к стандартному маркдауну или простому HTML. И только в самых сложных случаях — когда вот очень нужно — вводить замысловатые надстройки над маркдауном.

То есть сейчас я бы пошёл от задачи: какие атрибуты нам нужны настолько, что мы готовы усложнять?
